services/QuillLMS/app/controllers/api/v1/feedback_history_ratings_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::FeedbackHistoryRatingsController < ApplicationController
+  before_action :set_feedback_history_rating, only: [:show, :edit, :update, :destroy]
+  skip_before_action :verify_authenticity_token
+
+  def create_or_update
+    rating = FeedbackHistoryRating.find_or_create_by(
+      user_id: feedback_history_rating_params["user_id"], 
```

Comment:
Can we use `current_user` here instead of passing it as a param?

Reply:
Yes - I saw your comment above, but forgot to implement